integration/simulation/validate_chain.go

```diff
@@ -353,7 +358,7 @@ func extractWithdrawals(t *testing.T, nodeClient rpc.Client, nodeAddr uint64) (t
 			return
 		}
 		for _, w := range r.Withdrawals {
-			totalSuccessfullyWithdrawn += w.Amount
+			totalSuccessfullyWithdrawn.Add(totalSuccessfullyWithdrawn, w.Amount)
```

Comment:
Noob question, but don't you have to assign it here ?
`totalSuccessfullyWithdrawn = totalSuccessfullyWithdrawn.Add(totalSuccessfullyWithdrawn, w.Amount)` ?

Reply:
Apparently not and seems they recommend not to